packages/deployment/app/cloudformation.template.yml

```diff
@@ -16,6 +16,21 @@ Parameters:
     Default: NoValue
     NoEcho: true
 
+Mappings:
```

Comment:
this section should move to the config.yml, so it would be easier to for the template users to override it

Reply:
The intent was to keep a separation between SAAS and IAAS.  At the configuration level, the amiid should be defaulted to the best for AWS region.  This is really an IAAS concern, IMHO.  I can revert it back if you want.

Reply:
my point is that we can achieve the exact outcome even if we put this in the config.yml. It advantage is that you can override it easily when it is in the config.yml